article/_posts/2019-07-29-Dont-Worry-About-The-Weather.md

```diff
@@ -84,11 +85,11 @@ Given an input image $$I_A$$ and a domain label $$t$$, the goal is to find the p
 ![](/article/images/Weather/onlinelearning.png)
 
 The previous subsections can extend to incremental unsupervised, online learning of new unseen domains without requiring any significant modifications to the existing system.
-This processed used to:
-* Given a continuous sequence of incoming images, storing the current frame and $$T−1$$ past frames in a buffer of length $$T$$ that gets updated using a First-In-First-Out scheme.
-* For each frame in the buffer, a length-128 condition descriptor is compute using the penultimate layer of the classifier and average all the descriptors, yielding one single length-128 average descriptor.
-* If this average descriptor condition differs(in Euclidean space) by more than a threshold from the descriptors of any conditions previously trained on.
-the following training pipeline is triggered:
+This process is used to:
+* Give a continuous sequence of incoming images, storing the current frame and $$T−1$$ past frames in a buffer of length $$T$$ that gets updated using a First-In-First-Out scheme.
+* Compute each frame in the buffer using the penultimate layer of the classifier and average all the descriptors, yielding one single length-128 average descriptor.
+* Average descriptor condition differs(in Euclidean space) by more than a threshold from the descriptors of any conditions previously trained on.
```

Comment:
_Average descriptor conditions if they differ (in Euclidean space) by more than (...)_

Reply:
Changed in 9202b1c.